pypeit/specobjs.py

```diff
@@ -564,7 +564,9 @@ def write_to_fits(self, subheader, outfile, overwrite=True, update_det=None,
             outfile (str):
             overwrite (bool, optional):
             slitspatnum (:obj:`str` or :obj:`list`, optional):
-                Restricted set of slits for reduction
+              Restricted set of slits for reduction
+              If provided, do not clobber the existing file but only update
+              the indicated slits.  Useful for re-running on a subset of slits
```

Comment:
Nit-picking:  These descriptions need to read well regardless of the formatting.  I.e., sphinx will format this as it wants, not as you've put it here.  All this is a round about way of me asking you to add a period after `reduction` on the first line.

Reply:
:)